### ISSUE TITLE
Can't access content permissions via keyboard

### ISSUE THREAD

**Issue by thomasheartman** [enonic/app-contentstudio#3377]
When editing a piece of content, I am unable to open the permissions dialog using the keyboard only. The content ribbon contains elements that react to mouse clicks, but they're divs, meaning they're not focusable by keyboard. Arguably, they should be `<button>`s instead.

Here's the ribbon and the elements in question:

![deployment-open-permissions](https://user-images.githubusercontent.com/17786332/128172593-1ee6d913-2991-4949-ba17-d1e6b1a4766d.png)
